packs/spell-effects/spell-effect-life-link.json

```diff
@@ -0,0 +1,87 @@
+{
+    "_id": "8SkrekdJ9h3T664t",
+    "img": "systems/pf2e/icons/spells/life-link.webp",
+    "name": "Spell Effect: Life Link",
+    "system": {
+        "description": {
+            "value": "<p>Granted by @UUID[Compendium.pf2e.spells-srd.Item.Life Link]</p>\n<p>You forge a connection of life energy between you and your target, distributing harm between both of you.<br /><br /><strong>Apply Damage (Caster)</strong> @Damage[(3)[Hit Points]].</p>\n<p><strong>Heightened (3rd)</strong> @Damage[(5)[Hit Points]].</p>\n<p><strong>Heightened (6th)</strong>@Damage[(10)[Hit Points]].</p>\n<p><strong>Heightened (9th)</strong>@Damage[(15)[Hit Points]].</p>"
```

Comment:
We don't use `<br>` tags for newlines. Effects should also say what they do and nothing else.